templates/01-hello-username/package.json

```diff
@@ -12,21 +12,13 @@
     "test": "npm run build && jest test --env=near-shell/test_environment"
   },
   "devDependencies": {
-    "assemblyscript-json": "github:nearprotocol/assemblyscript-json",
-    "gh-pages": "^2.0.1",
-    "gulp": "^3",
-    "jest": "^22.4.4",
-    "jest-environment-node": "^24.5.0",
-    "near-runtime-ts": "github:nearprotocol/near-runtime-ts",
-    "near-shell": "github:nearprotocol/near-shell",
-    "bignum": "github:MaxGraey/bignum.wasm"
+    "assemblyscript-json": "github:WebAssemblyOS/assemblyscript-json#near_dist",
```

Comment:
@willemneal let's make sure everything depends on repos in nearprotocol, preferably `staging` branch

Reply:
I want to, but the staging branch on `assemblyscript-json` is still waiting for my PR and at the time of writing this it didn't exist.